### ISSUE TITLE
07_Inception_Model.ipynb give different results ?

### ISSUE THREAD

**Issue by shamanez** [Hvass-Labs/TensorFlow-Tutorials#37]
I went through this notebook with given introduction. But when I use **classify(image_path)** function in the notebook it doesn't give correct predictions all the time.
If I test it on the given panda image it will first predict correct labels. But if I run in again in will give results like  **website, web .. etc**

**Comment by Hvass-Labs** [comment]
I have just now tested this, and it works fine on my system which is Ubuntu 16.04, Python 3.6, TensorFlow 1.0. It gives the exact same outputs each time I call the function.

How do you expect me to help, when you don't give details about the system and software versions you are using, and the output / error messages you are seeing?

**Comment by shamanez** [comment]
I am sorry. My tensorflow version is 1.01. Ubuntu version is 16.04 and python version is 3.5.
When I run the notebook for several times in each instance it gives different predicted results. No error messages. 

This is the original result in the tutorial 
![1](https://cloud.githubusercontent.com/assets/16892570/25664296/d1bb296c-3037-11e7-9a65-9ba3ce1ccf29.png)

This is what I got as correct results
![2](https://cloud.githubusercontent.com/assets/16892570/25664334/eb72e700-3037-11e7-912d-75ea2725632d.png)

This is what I got after running the same cell which classify panda again 
![3](https://cloud.githubusercontent.com/assets/16892570/25664357/043611d6-3038-11e7-97a4-6998183acd5c.png)

**Comment by Hvass-Labs** [comment]
OK, that was a better error-report.

Unfortunately, I don't know what is wrong as it works fine for me. You will have to debug to find out what the problem is. It will be a good exercise for you to learn TensorFlow better. A wild guess might be that you have too little RAM, although that should probably give an error somewhere. You could try running it on a friend's PC and see if it works there.

If you find the solution then please report it here so it may help others in the future.

**Comment by shamanez** [comment]
Well no errors at all. Thanks a lot for the support. I will surely update if I found a solution. Well I think you are correct. I tried this with tensorflow tutorial on CNN. It happened the same. Different probabilities and answers.

**Comment by manselag123** [comment]
i pretrained the inception model in tutorial #09 on my dataset but i want to classify my images like the one in tutorial#07

**Comment by manselag123** [comment]
classify(myimage)

**Comment by Hvass-Labs** [comment]
@shamanez It is strange that the error happens when you rerun the entire Notebook and apparently gets worse when you just rerun a part of the Notebook. Try and download the Notebook and this entire repo again, perhaps it has been corrupted. Anyway, I can't help you so I'm closing this topic, but please leave a comment if you find the solution so it may help others in the future.

@manselag123 Your question is off-topic for this thread and I cannot help you with a question like this. You will have to do it yourself or ask on StackOverflow.

**Comment by shamanez** [comment]
@Hvass-Labs  Well it's not a problem with repo or the notebook. I tried the same with the tensorflow tutorial on image classification with tensorflow. (https://github.com/tensorflow/models)
It gave me the same errors. I think it's something to do with tf version I am using and the ram. Thanks a lot for responses.
